tests/modules/bismark/methylationextractor/test.yml

```diff
@@ -1,16 +1,48 @@
-- name: bismark methylation extractor test workflow
-  command: nextflow run ./tests/modules/bismark/methylationextractor -entry test_bismark_methylationextractor -c ./tests/config/nextflow.config -c ./tests/modules/bismark/methylationextractor/nextflow.config
+- name: bismark methylationextractor test_bismark_methylationextractor
+  command: nextflow run ./tests/modules/bismark/methylationextractor -entry test_bismark_methylationextractor -c ./tests/config/nextflow.config  -c ./tests/modules/bismark/methylationextractor/nextflow.config
   tags:
     - bismark
     - bismark/methylationextractor
   files:
+    - path: output/bismark/BismarkIndex/Bisulfite_Genome/CT_conversion/BS_CT.1.bt2
+      md5sum: 10a8854e5eeb95629f46d24b0d8da70c
+    - path: output/bismark/BismarkIndex/Bisulfite_Genome/CT_conversion/BS_CT.2.bt2
+      md5sum: bc5d407e6fce3e1cdb4b23ab2a20f707
+    - path: output/bismark/BismarkIndex/Bisulfite_Genome/CT_conversion/BS_CT.3.bt2
+      md5sum: 4ed93abba181d8dfab2e303e33114777
+    - path: output/bismark/BismarkIndex/Bisulfite_Genome/CT_conversion/BS_CT.4.bt2
+      md5sum: 6c4f549575e5882ff3a97ae10ae2e7be
+    - path: output/bismark/BismarkIndex/Bisulfite_Genome/CT_conversion/BS_CT.rev.1.bt2
+      md5sum: d37d586b41b6a36efb25839961d242cb
+    - path: output/bismark/BismarkIndex/Bisulfite_Genome/CT_conversion/BS_CT.rev.2.bt2
+      md5sum: 48bf400de85cd7324fbf6bf38fe2dc95
+    - path: output/bismark/BismarkIndex/Bisulfite_Genome/CT_conversion/genome_mfa.CT_conversion.fa
+      md5sum: 903b9f357eea4a5f36e21e78e0fe1dfa
+    - path: output/bismark/BismarkIndex/Bisulfite_Genome/GA_conversion/BS_GA.1.bt2
+      md5sum: a6fa4068ed10872568f32568c66cb600
+    - path: output/bismark/BismarkIndex/Bisulfite_Genome/GA_conversion/BS_GA.2.bt2
+      md5sum: fc5b634e2f5137801e3801d62ad05f74
+    - path: output/bismark/BismarkIndex/Bisulfite_Genome/GA_conversion/BS_GA.3.bt2
+      md5sum: 4ed93abba181d8dfab2e303e33114777
+    - path: output/bismark/BismarkIndex/Bisulfite_Genome/GA_conversion/BS_GA.4.bt2
+      md5sum: b1e855685ed634daa7df8e180c9b2fc4
+    - path: output/bismark/BismarkIndex/Bisulfite_Genome/GA_conversion/BS_GA.rev.1.bt2
```

Comment:
where did all these bt2 files come from?

To regenerate this test.yml run
```
nf-core modules create-test-yml bismark/methylationextractor
```

It would also be beneficial to have a two test workflows. One for single-end data and another for paired-end data. Perhaps use the single-end case to test with minimal args, and the paired-end test to test with all args that produce output files into each channel.